Dockerfile.Toil

```diff
@@ -29,10 +29,9 @@ RUN apt-get install -y docker-ce
 
 #install nodejs, to provide in line javascript evaluation
 # RUN apt-get install -y nodejs npm
-RUN curl -sL https://deb.nodesource.com/setup_6.x | sudo bash -
+RUN curl -sL https://deb.nodesource.com/setup_14.x | sudo bash -
 RUN apt-get install -y nodejs
-RUN apt-get install -y npm
-
+# RUN apt-get install -y npm
```

Comment:
Are you sure this isn't needed?  I added it to be able to parse in-line javascript in .cwl files.

Reply:
It's already installed by installing nodejs with the later version of nodejs. I will do some further testing to confirm.

Reply:
I tested and it works without this line.  `npm` is actually installed already. (See description)

Reply:
OK, then may I ask that you remove the commented line altogether?  We always try not to have "dead code" in our code base.